hw/ip_templates/gpio/dv/env/gpio_agent/gpio_data_agent.sv.tpl

```diff
@@ -0,0 +1,45 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+class gpio_data_agent extends dv_base_agent #(
```

Comment:
This filename will be removed to `module_instance_name_data_agent.sv`, I think the module name need to adapt as well.